src/sagemaker_xgboost_container/algorithm_mode/serve.py

```diff
@@ -35,6 +35,19 @@
 logging = integration.setup_main_logger(__name__)
 
 
+def _get_max_content_size():
```

Comment:
Is there a reason this uses the word "size" when everything else refers to "length"?